<launch>
  <node name="shoot_navigation_multi_goals" pkg="robot_slam" type="shoot_navigation_multi_goals.py" output="screen">
      <param name="goalListX" value="  1.15 ,   1.12,   1.12,   0.35,   0.14 ,0.14"/>
      <param name="goalListY" value=" -0.39, -1.51, -2.77,  -2.95,   -2.28,-1.1"/>
      <param name="goalListYaw" value="0.0, 0.0, 0.0, 0.0, 0,0 "/>
  </node>
</launch>
